test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/Util.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.XPlat.FuncTest
+{
+    internal class Util
+    {
+        private static string DotnetCliBinary { get; set; }
+
+        private static string DotnetCliExe { get; set; }
+
+        private static string XplatDll { get; set; }
+
+        private static string XplatDllShell { get; set; }
+
+        static Util()
```

Comment:
No need for a static ctor or properties here, just set these strings to be `private const string`.